### DISCUSSION TITLE
Best way to customize

### DISCUSSION THREAD

**Discussion by alexandremjacques** [tabler/tabler#1260] [Q&A]
Hi,

What should be the correct way to customize Tabler? I saw somewhere about customizing fonts but my need would be to, mainly, customize colors.

TIA

**Comment by alexandremjacques** [comment]
I've tried altering the `demo.scss` file and rebuild (`npm build`) the project but I can't see the changes of the `$primary` color I have set reflected in the demo site generated.

Would this be the correct path?

**Comment by cavasinf** [comment]
I've come across the same problem/question.

I ended creating a new scss file.
You can check the doc here : https://github.com/kevinpapst/TablerBundle/blob/main/docs/override_theme.md

**Reply by alexandremjacques** [reply]
Thanks!

But, where this file goes in the Tabler directory structure? Can't seem to find a way to override the defaults unless I change the `.scss` include files.

**Reply by cavasinf** [reply]
You can download the [`theme.scss` file](https://github.com/kevinpapst/TablerBundle/blob/main/docs/theme.scss) and import it just after your [`tabler.css`](https://github.com/tabler/tabler/blob/dev/dist/css/tabler.css) import HTML script.

**Reply by alexandremjacques** [reply]
Doesn't it have to be "compiled" Sass since it's a `.scss` file? I did create a `theme.scss` file under `src/scss` folder. It gets compiled ok and generates a `theme.css` file under `demo/dist/css`. 

However, it doesn't override the items I put in it. I even changed `demo/index.html` file to use it.

```
<link href="./dist/css/theme.min.css" rel="stylesheet"/>
```

Nothing seems to work.

**Reply by cavasinf** [reply]
It generates `theme.css` or `theme.min.css` ?

If I take example with the demo empty page => https://github.com/tabler/tabler/blob/dev/demo/empty.html

It should look like this:
```diff
<head>
    <meta charset="utf-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"/>
    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
    <title>Empty page - Tabler - Premium and Open Source dashboard template with responsive and high quality UI.</title>
    <!-- CSS files -->
    <link href="./dist/css/tabler.min.css" rel="stylesheet"/>
    <link href="./dist/css/tabler-flags.min.css" rel="stylesheet"/>
    <link href="./dist/css/tabler-payments.min.css" rel="stylesheet"/>
    <link href="./dist/css/tabler-vendors.min.css" rel="stylesheet"/>
    <link href="./dist/css/demo.min.css" rel="stylesheet"/>
+   <link href="./dist/css/theme.css" rel="stylesheet"/>
    <style>
      @import url('https://rsms.me/inter/inter.css');
      :root {
      	--tblr-font-sans-serif: Inter, -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif;
      }
    </style>
  </head>
```

**Reply by cavasinf** [reply]
I've just updated the file.
Get the new one here : https://github.com/kevinpapst/TablerBundle/blob/main/docs/theme.scss
